packages/os/os.spec

```diff
@@ -514,6 +545,15 @@ install -p -m 0644 %{S:300} %{buildroot}%{_cross_udevrulesdir}/80-ephemeral-stor
 %{_cross_bindir}/static-pods
 %endif
 
+%if %{_is_vmware_variant} || %{_is_dev_variant}
+%files -n %{_cross_os}prairie-dog
+%{_cross_bindir}/prairie-dog
+%{_cross_unitdir}/kdump.service
+%{_cross_unitdir}/kexec.service
+%{_cross_unitdir}/configure-boot-mount.service
+%{_cross_tmpfilesdir}/kdump.conf
+%endif
```

Comment:
I'd prefer to unconditionally build and include `prairiedog` and decide at runtime whether to do anything based on the presence of `crashkernel` in the command line.

Possibly this could use the `ConditionKernelCommandLine=` test that systemd offers.